README.md

```diff
@@ -311,6 +315,32 @@ sudo ln -s $(which rustc)
 
 This step needs to be executed each time Xcode updates.
 
+#### Building using Conda on MacOS
+
+We provide a Conda env for easy Setup. 
+Prerequisites: 
+- Have miniconda installed. 
+
+```bash 
+$ conda env create -f env.yml
+$ conda activate VPN
+```
+ - Get a copy of a MacOS-SDK (every X-Code install ships this, or you can find it on the internet :) ) 
+    - set `SDKROOT` to the target SDK.
+    - Suggested to add it to the conda env via: `conda env config vars set SDKROOT=<>`
```

Comment:
nit:
```suggestion
    - Add it to the conda env via: `conda env config vars set SDKROOT=<>`
```
Just because we are being imperative in all other bullet points.